src/ctapipe/monitoring/interpolation.py

```diff
@@ -4,15 +4,13 @@
 import astropy.units as u
 import numpy as np
 import tables
+from astropy.table import Table
 from astropy.time import Time
 from scipy.interpolate import interp1d
 
 from ctapipe.core import Component, traits
 
-__all__ = [
-    "Interpolator",
-    "PointingInterpolator",
-]
+__all__ = ["PointingInterpolator", "ChunkInterpolator"]
```

Comment:
Please do not remove the base `Interpolator` class here, it will prevent the interface from being documented in the sphinx docs

Reply:
Ok, i'll do it while adding the changelog